rust_src/Cargo.toml

```diff
@@ -58,9 +58,7 @@ panic = "abort"
 
 [features]
 default = [
-  "use-xml2", "unexec", "window-system", "window-system-x11", "libvterm",
-
-]
+  "use-xml2", "unexec", "window-system", "window-system-x11",]
```

Comment:
Do we want to remove libvterm here intentionally?

Reply:
Yeah, it's currently not in the code base.